LookerEmbedClientExample.java

```diff
@@ -38,7 +38,7 @@ public static void main(String [] args){
 
     public static String createURL(String host, String secret,
                                    String userID, String firstName, String lastName, String userPermissions,
-                                   String userModels, String sessionLength, String accessFilters,
+                                   String userModels, String sessionLength, String groupIds, String accessFilters,
```

Comment:
Minor nit: How will the addition impact existing callers?  Compiler error msg about "missing parameter forceLoginLogout" or somesuch.  If the new param were added at the end of the param list, compiler message would be accurate.  esp. true when all params are the same type.